recipes-bsp/alsa-state/alsa-state.bbappend

```diff
@@ -1 +1,15 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append_hardkernel-odroidn2plus = " file://soundconfig \
+                                           file://90-alsa-restore.rules \
+                                         "
+
+FILES_${PN}_append_hardkernel-odroidn2plus = " ${libdir}/udev/soundconfig \
+                                               ${libdir}/udev/rules.d/90-alsa-restore.rules \
+                                             "
+
+do_install_append() {
+    install -d ${D}/${libdir}/udev/rules.d
+    install soundconfig ${D}/${libdir}/udev
+    install 90-alsa-restore.rules ${D}/${libdir}/udev/rules.d
+}
```

Comment:
This should probably be a bbappend to alsa-lib instead

Reply:
And replace the alsa-lib `90-alsa-restore.rules` like does LibreELEC

Reply:
But the udev rule runs script which depends on amixer, so perhaps it should go to something that depends on amixer at least implicitly and alsa-lib doesn't.
LibreELEC does this one: https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/audio/alsa-utils/package.mk
With yocto we don't need to care about it right?
Please let me know where you would like to see those scripts. Perhaps we should add alsa-default-state  package which can be empty for machines that are not handled by soundconfig script. What do you think?

Reply:
Perhaps, but you'll still have conflict with the udev-rules.